conda_forge_tick/hashing.py

```diff
@@ -120,6 +120,7 @@ def hash_url(url, timeout=None, progress=False, hash_type="sha256"):
             raise e
 
     if isinstance(_hash, tuple):
+        # TODO: What is this? (remove noqa from above)
```

Comment:
@beckermr What happens here? Is this a bug?

Reply:
I don't know. It's been a long time.

Reply:
I bet one of the cases in the try except above produces a tuple.

Reply:
Yeah, but I don't get why it's evaled

Reply:
same

Reply:
Okay then let's keep this with noqa for now